app/frontend/components/pages/dashboard/walletOperationStatus.tsx

```diff
@@ -25,19 +46,23 @@ export const WalletOperationStatusType = () => {
   switch (walletOperationStatusType) {
     case 'txPending':
       return (
-        <Fragment>
-          <PinkSpinner message={'Transaction pending...'} />
-          <p className="loading-helper-text">
-            Taking too long? Read why{' '}
-            <a
-              href="https://github.com/vacuumlabs/adalite/wiki/Troubleshooting#my-transaction-has-been-pending-for-a-long-time--what-should-i-do"
-              target="_blank"
-            >
-              here
-            </a>
-            .
-          </p>
-        </Fragment>
+        <PinkSpinner
+          message={'Transaction pending...'}
+          helperText={
```

Comment:
Seems easier to add helper text absolutely to PinkSpinner as it has hardcoded margin values for responsive designs.